scripts/report-test-results.py

```diff
@@ -0,0 +1,110 @@
+import os
+import sys
+from xml.dom.minidom import Node, parse
+
+
+def main():
+    if len(sys.argv) < 3:
+        print("Usage: python report-test-results.py working-dir testsuite-list")
+        sys.exit(0)
+
+    working_dir = sys.argv[1]
+    test_file_names = sys.argv[2:]
+    test_files = [
+        working_dir + "/functional-test-reports/" + filename
+        for filename in test_file_names
+    ]
+
+    if not len(test_files):
+        print("Please provide a list of test files")
+        sys.exit(0)
+
+    artefact_bucket = get_smoke_test_bucket_name()
+
+    for test_file in test_files:
+        test_results = extract_test_result(parse(test_file))
+        for r in test_results:
+            success = "PASS" if len(r) < 4 else "FAIL"
```

Comment:
Is this right? If I follow, it's considered a pass if not everything failed (i.e., if 3 failed, this would be a pass)? Okay if expected, just thought it would be worth double checking

Reply:
There are at least three elements to the result  [GROUP, NAME, TIME, ERROR, FILE], only the first three are populated if the test passed. If we see more than three elements in this array, then we know that test failed.

Reply:
Ahh, I see, makes sense. Sounds fair enough then